Hackney.Shared.Tenure/Boundary/Requests/Validation/TenureInformationValidatorWhenOnlyEndDate.cs

```diff
@@ -16,7 +16,7 @@ public TenureInformationValidatorWhenOnlyEndDate()
 
             // the end date must be greater than start date
             RuleFor(x => x.EndOfTenureDate)
-            .GreaterThan(x => x.StartOfTenureDate)
+            .GreaterThanOrEqualTo(x => x.StartOfTenureDate)
```

Comment:
And this:

https://github.com/LBHackney-IT/tenure-shared/blob/61af1209476be3cccc3c78568f4086aefae5383e/Hackney.Shared.Tenure/Boundary/Requests/Validation/CreateTenureRequestValidation.cs#L11

Could you take a look through all the validators that are validating start and end dates?

Reply:
I've now (hopefully) changed this on all validators.